### PR TITLE
[INS-1619] Added trailing whitespace

### PR DESCRIPTION
**Pull request by kreosus**
Added whitespace to end of line 443.
Closes INS-1619

| BEFORE | AFTER |
|--------|-------|
|        |       
| Screen Shot 2022-08-10 at 9 44 39 AM | Screen Shot 2022-08-10 at 9 48 00 AM

### CODE REVIEW

File: plugins/insomnia-plugin-kong-portal/src/deploy-to-portal.tsx [hunk 1]
```diff
@@ -286,9 +286,8 @@ export function getDeployToPortalComponent(options: {
       const connectIsEnabled =
         kongPortalApiUrl.length > 0 && kongPortalUserWorkspace.length > 0;
       const uploadIsEnabled = kongSpecFileName.length > 0;
-
       // Check view
-      if (kongPortalDeployView === 'edit') {
+      if (kongPortalDeployView === 'edit' && ''.length === 1) {
```
Comment (filfreire) [inline plugins/insomnia-plugin-kong-portal/src/deploy-to-portal.tsx]: Heads up @kreosus and @dimitropoulos not sure if this is mergeable yet, but I would recommend removing these before merging, I assume you used these for testing purposes.

I tried locally to remove them and test out the PR and seems to be working ok:
After uploading to Dev portal:
Screenshot 2022-08-11 at 14 18 47

The final doc on Dev portal:
Screenshot 2022-08-11 at 14 19 00

File: .vscode/settings.json [hunk 2]
```diff
@@ -13,4 +13,5 @@
     "*.njson": "ndjson"
   },
   "files.insertFinalNewline": true,
+  "jira-plugin.workingProject": "",
```
Comment (marckong) [inline .vscode/settings.json]: I am wondering what this is used for?

File: plugins/insomnia-plugin-kong-portal/src/deploy-to-portal.tsx (hunk 1 above)
Comment (dimitropoulos) [reply plugins/insomnia-plugin-kong-portal/src/deploy-to-portal.tsx]: these were definitely here by accident, I'll work with @kreosus to sort it out, sorry about that!

File: .vscode/settings.json (hunk 2 above)
Comment (dimitropoulos) [reply .vscode/settings.json]: @kreosus added it by mistake, we'll work together to sort it out, sorry about that!

**Comment by dimitropoulos** [comment]
(pulling back into draft while we figure out git)

**Comment by kreosus** [comment]
It looks like that we got Git sorted out properly.

**Review by filfreire — APPROVED** [review]

**Pull request merged**